server/chainquery/index.js

```diff
@@ -0,0 +1,80 @@
+const Sequelize = require('sequelize');
+const logger = require('winston');
+
+import abnormalClaimTable from './tables/abnormalClaimTable';
```

Comment:
Is this useful to you? This is for claims that were not made through the app. Claims that are not structured according to the lbry protocol.

Reply:
Not at the moment, retained for full functionality if this is to be split out into a separate lib